tools/ci_build/github/azure-pipelines/qnn-ep-nuget-packaging-pipeline.yml

```diff
@@ -7,7 +7,7 @@ parameters:
 - name: build_config
   displayName: Build Configuration
   type: string
-  default: 'Release'
+  default: 'RelWithDebInfo'
```

Comment:
why change from Release to RelWithDebInfo as the default?

Reply:
Other jobs in the big job Zip-Nuget-Java-Nodejs Packaging Pipeline all use RelWithDebInfo.